packages/building-webpack/default-config.js

```diff
@@ -0,0 +1,168 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { BabelMultiTargetPlugin } = require('webpack-babel-multi-target-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+
+const ENV = process.argv.find(arg => arg.includes('production')) ? 'production' : 'development';
+const developmentEs5 = !!process.argv.find(arg => arg.includes('es5'));
+
+const defaultOptions = {
+  indexHTML: './index.html',
+  indexJS: ['./index.js'],
+};
+
+const es6Target = {
+  browsers: [
+    'last 2 Chrome major versions',
+    'last 2 ChromeAndroid major versions',
+    'last 2 Edge major versions',
+    'last 2 Firefox major versions',
+    'last 3 Safari major versions',
+    'last 3 iOS major versions',
+  ],
+  tagAssetsWithKey: false,
+  esModule: true,
```

Comment:
while playing around with just webpack I found out that combining browsers + esmodule is not allowed at least babel shows a warning for it (one of the reasons I'm in doubt of the plugin now as it apparently suppresses these messages...)
> Please note: when specifying the esmodules target, browsers targets will be ignored.

https://babeljs.io/docs/en/babel-preset-env#targetsesmodules

=> makes sense either you say work for these browsers or work for all browsers that support es modules

Reply:
This is not the preset-env config, but the multi target config. The `esModule` option here means whether it should be loaded as `type="module"`